activities/views/posts.py

```diff
@@ -39,42 +39,30 @@ def get(self, request, handle, post_id):
             # Show normal page
             return super().get(request)
 
-    def get_context_data(self):
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context["opengraph"] = self.post_obj.to_opengraph_dict()
```

Comment:
This is not needed, `post.to_opengraph_dict` is usable within the template.

Reply:
Yes, but I don't know that I need to call `post` or `identity`, or some future object, or the detault OG keyset, so I need some kind of indirection.

Reply:
Oh, I see your other comment. Looking.